runtime/fundamentals/testing.md

```diff
@@ -97,7 +96,7 @@ deno test my_test.ts -- -e --foo --bar
 
 # Provide permission for deno to read from the filesystem, which is necessary
 # for the final test above to pass
-deno test --allow-read my_test.ts
+deno test --allow-read=. my_test.ts
```

Comment:
This is a bit of a personal preference, but I have a strong desire to promote the practice of not granting broad permissions too easily.